wifi-heatmap.py

```diff
@@ -1,6 +1,6 @@
 from __future__ import division
 
-import tabular as tb
+import pandas as pd
```

Comment:
It's best to list the external dependencies in something like `requirements.txt`, at least.

Reply:
Added requirements.txt for external dependencies